lib/events/api.go

```diff
@@ -100,6 +100,9 @@ const (
 	// SessionStartTime is the timestamp at which the session began.
 	SessionStartTime = "session_start"
 
+	// SessionRecordingType is the type of session recording
+	SessionRecordingType = "session_recording"
```

Comment:
This is missing comments on what values are possible, presuming this is a string-style enumeration.